code/_onclick/click.dm

```diff
@@ -246,12 +246,15 @@
 	A.CtrlClick(src)
 	return
 /atom/proc/CtrlClick(var/mob/user)
+	if("pull")
+		usr.stop_pulling()
```

Comment:
user, not usr.
The if also does nothing.
I'd move this under the AM CtrlClick, and use
`else if(user.pulling == src)
   user.stop_pulling()`